content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -65,72 +124,161 @@ resources:
+`aescbc` | AES-CBC with [PKCS#7](https://datatracker.ietf.org/doc/html/rfc2315) padding | Weak | Fast | 32-byte | Not recommended due to CBC's vulnerability to padding oracle attacks. Key material accessible from control plane host.
+`aesgcm` | AES-GCM with random nonce | Must be rotated every 200,000 writes | Fastest | 16, 24, or 32-byte | Not recommended for use except when an automated key rotation scheme is implemented. Key material accessible from control plane host.
+`secretbox` | XSalsa20 and Poly1305 | Strong | Faster | 32-byte | Uses relatively new encryption technologies that may not be considered acceptable in environments that require high levels of review. Key material accessible from control plane host.
+`kms` | Uses envelope encryption scheme: Data is encrypted by data encryption keys (DEKs) using AES-CBC with [PKCS#7](https://datatracker.ietf.org/doc/html/rfc2315) padding (prior to v1.25), using AES-GCM starting from v1.25, DEKs are encrypted by key encryption keys (KEKs) according to configuration in Key Management Service (KMS) | Strongest | Fast | 32-bytes |  The recommended choice for using a third party tool for key management. Simplifies key rotation, with a new DEK generated for each encryption, and KEK rotation controlled by the user. [Configure the KMS provider](/docs/tasks/administer-cluster/kms-provider/)
+{{< /table >}}
+
+By default, the `identity` provider is used to protect data in etcd, which provides **no**
+encryption. `EncryptionConfiguration` was introduced to encrypt confidential data locally, using
+a locally managed key.
 
-Each provider supports multiple keys - the keys are tried in order for decryption, and if the provider
-is the first provider, the first key is used for encryption.
+Encrypting secret data with a locally managed key protects against an etcd compromise, but it fails to
+protect against a host compromise. Since the encryption keys are readable - in the clear - on the
+control plane host in the EncryptionConfiguration YAML file, a skilled attacker can access that file
+and extract the encryption keys.
+
+Envelope encryption creates dependence on a separate key, not stored in Kubernetes. In this case,
+an attacker would need to compromise etcd, the `kubeapi-server`, and the third-party KMS provider to
+retrieve the plaintext values, providing a higher level of security than locally stored encryption keys.
 
+## Encrypt your data {#encrypting-your-data}
 
 {{< caution >}}
-Storing the raw encryption key in the EncryptionConfig only moderately improves your security
-posture, compared to no encryption.  Please use `kms` provider for additional security.
+Storing the raw encryption key in the EncryptionConfig only moderately improves your security posture,
+compared to no encryption.
```

Comment:
It can be OK to make the same point more than once in a document. See (eg) [The Use of Repetition in Technical Communication](https://kmh-lanl.hansonhub.com/pc-26-9-zimmerman.pdf).

I think this is one of those occasions. If someone has skipped to this heading, it's still worth showing them this admonition.